addons/payment_stripe/models/payment_token.py

```diff
@@ -0,0 +1,75 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+import logging
+
+from odoo import fields, models, _
+from odoo.exceptions import ValidationError
+
+_logger = logging.getLogger(__name__)
+
+
+class PaymentToken(models.Model):
+    _inherit = 'payment.token'
+
+    stripe_payment_method = fields.Char('Payment Method ID')
+
+    def _get_specific_create_values(self, provider, values):
+        if provider != 'stripe':
+            return
+        if not values.get('acquirer_ref'):
+            partner_id = self.env['res.partner'].browse(values.get('partner_id'))
+            payment_acquirer = self.env['payment.acquirer'].browse(values.get('acquirer_id'))
```

Comment:
Keep short and clear variables (only use `_id` for real ids, not for recordsets)
```suggestion
            partner = self.env['res.partner'].browse(values.get('partner_id'))
            acquirer = self.env['payment.acquirer'].browse(values.get('acquirer_id'))
```